static/scripts/loggedin.js

```diff
@@ -66,42 +66,30 @@ $(document).ready(function () {
         $.showNotification(message, "success", true);
     }
 
-    /**
-     * creates the feedback-message which will be sent to the Schul-Cloud helpdesk
-     * @param modal {object} - modal containing content from feedback-form
-     */
-    const createFeedbackMessage = function (modal) {
-        return "Als " + modal.find('#role').val() + "\n" +
-            "möchte ich " + modal.find('#desire').val() + ",\n" +
-            "um " + modal.find("#benefit").val() + ".\n" +
-            "Akzeptanzkriterien: " + modal.find("#acceptance_criteria").val();
-    };
-
     const sendFeedback = function (modal, e) {
         e.preventDefault();
 
         let type = (modal[0].className.includes('feedback-modal')) ? 'feedback' : 'problem';
```

Comment:
Please refactor any code containing "feedback" and "problem" because the types changed, for better understanding. It is more like "contactHPI" and "contactAdmin" now.